### PR TITLE
✨ Add share buttons

### PR DESCRIPTION
**Pull request by AuroraHuang22**

### CODE REVIEW

File: src/pages/nft/class/_classId.vue [hunk 1]
```diff
@@ -294,6 +300,17 @@ export default {
...
       }
     },
+    handleCopyURL() {
+      const host = `${window.location.protocol}//${window.location.host}`;
+      const { path } = this.$route;
+      const URL = `${host}${path}?referrer=${this.getAddress}`;
+
+      copyURL(URL);
+
+      logTrackerEvent(this, 'ShareNFTDetails', 'CopyShareURL', URL, 1);
```
Comment (nwingt) [inline src/pages/nft/class/_classId.vue:310]:
```suggestion
      logTrackerEvent(this, 'NFT', 'CopyShareURL(Details)', URL, 1);
```

File: src/pages/nft/class/_classId.vue [hunk 2]
```diff
@@ -294,6 +300,17 @@ export default {
         this.alertPromptError('LIKE_PRICE_IS_TEMPORARY_UNAVAILABLE');
       }
     },
+    handleCopyURL() {
+      const host = `${window.location.protocol}//${window.location.host}`;
+      const { path } = this.$route;
+      const URL = `${host}${path}?referrer=${this.getAddress}`;
```
Comment (nwingt) [inline src/pages/nft/class/_classId.vue:306]: URL is a class 😂, might get confused
```suggestion
      const url = `${host}${path}?referrer=${this.getAddress}`;
```

File: src/util/ui.js [hunk 3]
```diff
@@ -24,4 +24,18 @@ export function ellipsisDescription(value) {
   return value;
 }
 
+export function copyURL(text) {
```
Comment (nwingt) [inline src/util/ui.js:27]:
```suggestion
export function copyToClipboard(text) {
```

**Review by WeiJun0827 — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

**Review by nwingt — COMMENTED** [review]
Seems `<ClipBoard>` is better to name as `<ShareButton>`?

**Pull request merged**